stdlib/importlib/readers.pyi

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Not a matter for this PR, but at runtime (at least on main) this file re-exports code from `importlib.resources.readers`, so ideally we should do the same in our stub.

Reply:
Yeah, see the comments on lines 1-3 of this file :)